### DISCUSSION TITLE
How to get OMB as default?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [ohmybash/oh-my-bash#614] [Q&A] [answered]
The other week or so i changed to fish with chsh, but right now i cannot find how to get back to omb as default shell (without having to remove all files and re-run the install)

**Comment by akinomyoga** [comment, marked as answer]
You can simply switch back to Bash using `chsh -s /path/to/bash` (please replace `/path/to` with the actual location of your Bash found in `/etc/shells`). OMB is not a shell, but a configuration for Bash. If you haven't touched the Bash settings while trying Fish, OMB should still be loaded by Bash through your Bash settings at `~/.bashrc`.

**Reply by GronbergInfo** [reply]
Thanks, that did it.
:D
